### ISSUE TITLE
[RFC] Configuration of edi

### ISSUE THREAD

**Issue by etobella** [OCA/edi#389]
On my experience making edi  I have found that is hard to define how to manage the configuration for a partner or other entities. Let me try to explain it:

Let's take accounting. A partner, could generate multiple integrations through edi, for a example, the invoice file using a SFTP, a notification using a WS and notification to Tax Agency. Obviously, each one is generated for different reasons (maybe two by partner, and one for journal?) Also, the configuration might be different by company (different storages or identifications). I was thinking on doing a new model called `edi.configuration` that will contain:

* model
* res_id
* backend_id
* company_id
* exchange_type_id
* parameters data (maybe a yaml?)
* Maybe some fields to detect when to use it

WDYT?

Should we include this on `edi` / `edi_oca` or create a new module just for this?

@simahawk

**Comment by bealdav** [comment]
Hi @etobella you're probably right to define these configuration in a dedicated model.
Maybe we could use this kind of field https://www.odoo.com/documentation/14.0/developer/reference/orm.html#odoo.fields.Reference
to help in a generic approach.

**Comment by simahawk** [comment]
@etobella this is something that should replace the rules we have already in the core (model_ids, active_domain, etc)?
I'm favour of a new model but I think we should put in a separate module.
Before starting w/ it could you summarize here what would you expect?
For instance:a re these rules for auto-generating exchanges? Or only for UI purposes? How should they kick-in in current machinery?
